### PR TITLE
Update 5.x.x Stable node to 5.5.0

### PR DESCRIPTION
**Pull request by kgalli**
Please read the title :-) ...

### CODE REVIEW

**Comment by prayerslayer** [comment]
These versions change too fast :grin: Will look into it on Monday, ok?

**Comment by prayerslayer** [comment]
I'm wondering if we should change the folder names to something along the lines of `stable`, `lts` and `ancient`?

File: 5.5/Dockerfile [hunk 1]
```diff
@@ -14,12 +14,14 @@ RUN set -ex \
...
+    B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+    C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
   ; do \
     gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 5.4.1
+ENV NODE_VERSION 5.5.0
 ENV NPM_VERSION 3.5.3
```
Comment (prayerslayer) [inline 5.5/Dockerfile]: npm is at `3.7.1` now, we can bump this as well, I think.

**Comment by kgalli** [comment]
Yap I will adjust the NPM version. And yes we should rename the folders to more generic names. I can create another PR for the renaming right after this PR and the LTS PR update is merged. What do you think?

**Comment by kgalli** [comment]
Update of NPM is done.

**Comment by prayerslayer** [comment]
Yeah that would be cool :+1:

**Comment by prayerslayer** [comment]
Oh, btw: For this release I'll have to reconfigure our Go CD pipeline, so it might take one or two days until it's available.

**Comment by prayerslayer** [comment]
Released as `5.5-11`.

**Comment by kgalli** [comment]
:+1:

**Pull request merged**